src/components/editTestCase/EditTestCase.test.tsx

```diff
@@ -1122,13 +1108,11 @@ describe("EditTestCase component", () => {
     });
 
     userEvent.click(screen.getByTestId("details-tab"));
-    expect(
-      await screen.findByTestId("create-test-case-title")
-    ).toBeInTheDocument();
+    expect(await screen.findByTestId("test-case-title")).toBeInTheDocument();
     // await waitFor(
```

Comment:
We should probably just delete this commented code.

Reply:
Agree I am not sure why it was commented on in the first place. So I didn't bother. let me take a look at that test case real quick

Reply:
fixed the test case